shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/UTxO.hs

```diff
@@ -220,19 +219,25 @@ balance (UTxO utxo) = foldr addCoins 0 utxo
   where
     addCoins (TxOut _ a) b = a + b
 
--- | Determine the total deposit amount needed
+-- | Determine the total deposit amount needed.
+-- Since this list may contain multiple (RegPool hk) certificates for the
```

Comment:
Which list?

Reply:
I would like to expand on the documentation here. Something like:

The block may (legitimately) contain multiple registration certificates for the same pool, where the first will be treated as a registration and any subsequent ones as re-registration. As such, we must only take a deposit for the first such registration.

Note that this is not an issue for key registrations since subsequent registration certificates would be invalid.

Reply:
thanks, I've used the above instead